src/frontend/next/src/pages/_app.tsx

```diff
@@ -1,17 +1,28 @@
 import { useEffect, useState } from 'react';
 import { AppProps } from 'next/app';
 import { ThemeProvider } from '@material-ui/core/styles';
+
 import Header from '../components/Header';
 import UserProvider from '../components/UserProvider';
 import '@fontsource/roboto';
 import '@fontsource/spartan';
 
 import '../styles/globals.css';
+
 import { darkTheme, lightTheme } from '../theme';
+import usePreferredTheme from '../hooks/use-preferred-theme';
+import { ThemeContext } from '../components/ThemeProvider';
+
+import '../styles/globals.css';
```

Comment:
This is already imported, so you can remove it

Reply:
Line 9 x line 14.

Reply:
I removed one already. Initially there were two duplicate imports . Hence for the lint error